blockchain/My-Wallet-V3-iOS
# feat(HB): Conversion Rates View - IOS-1208

## Objective

This adds in the view for displaying the latest conversion rates. 

## Description

Once a user enters in a value into the exchange screen, we get conversion rates. These rates can be displayed by tapping the rates view or the button within the rates view. 

**Note: We need to execute a conversion immediately upon showing this screen rather than only after a user enters a value. This isn't in this PR. The reason we need this is we need the rates immediately**

## How to Test

1. Build and run
2. Go to HB exchange
3. Enter in a value to execute a trade
4. Tap the rates view or rates button to see the conversion rates
5. The conversion rates should show

**Note: The animation isn't "expanding", it does animate though. I can go back and polish this up to give it an expansion animation. I'm prioritizing functionality for now.**

## Screenshot/Design assessment

![simulator screen shot - iphone x - 2018-09-18 at 14 39 18](https://user-images.githubusercontent.com/41585563/45712257-72594c00-bb51-11e8-877a-558529714189.png)

Design is at 90% given that the "expansion" animation isn't there. It's possible we won't have this style of animation since the keypad no longer has borders around the buttons. 

## Merge Checklist
